repos/system_upgrade_supplements/common/actors/checkreboothygiene/actor.py

```diff
@@ -21,7 +19,7 @@ class CheckRebootHygiene(Actor):
     name = "check_reboot_hygiene"
     consumes = ()
     produces = (Report,)
-    tags = (ChecksPhaseTag,)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
```

Comment:
The IPUWorkflowTag proved to be needed if we want to run the actor during the standard "Checks" phase of the IPU workflow